### ISSUE TITLE
ProgrammingError when using Faker

### ISSUE THREAD

**Issue by GitRon** [RegioHelden/django-scrubber#12]
* Django Scrubber version: 0.3.1
* Django version: 2.2.6
* Python version: 3.6
* Operating System: Win10

My setup:
```
class MyModel:
    my_bool = models.BooleanField(default=False)

    class Scrubbers:
        my_bool = scrubbers.Faker('pybool')
```

Result (translated from German):

> django.db.utils.ProgrammingError: ERROR:  Column »my_bool« has typ boolean, but expression has type character varying
LINE 1: UPDATE "app_mymodel" SET "my_bool" = (SELECT U0."...

Any ideas why this is happening? I tried the `boolean` provider as well and the same happens for `date_object`.

Best and thanks  
Ronny

**Comment by costela** [comment]
Hi!

You probably figured this out already, but just for future reference: this is related to https://github.com/RegioHelden/django-scrubber/pull/11#issuecomment-433073220 which requires some refactoring.

Since we don't require this feature internally, it was somewhat forgotten, but it's definitely useful and should get some attention. Realistically though, I'm just not sure I'll ever get around to it :confused: 

Having said that, your specific case doesn't seem particularly important: the cardinality of a `bool` field is small enough for it's privacy implications to be insignificant. Unless you have a big number of bool fields that can somehow be combined to effectively deanonymize the scrubbed entry, it doesn't seem like `django-scrubber` would be very helpful.
Is there maybe some context to your problem I might be missing?

**Comment by GitRon** [comment]
Hi @costela 

thanks for the reply! 

Maybe you are right about the booleans, but dates do contain some information. Image a guy having his birthday on Feb 29th and so on.

I agree on your post, I think there must be a mapping which casts the providers from faker to the correct value.

Maybe you shoud at least point this out on the docs page?

Best  
Ronny

**Comment by costela** [comment]
Should be fixed by #11.  
Release coming shortly.
